tests/unit/test_utils.py

```diff
@@ -0,0 +1,22 @@
+"""
+Copyright (c) 2022 Aiven Ltd
+See LICENSE for details
+"""
+from pathlib import Path
+from tests.utils import get_clickhouse_version
+from typing import List, Union
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.mark.parametrize(
+    "command,output,expected",
+    [
+        ([Path("/usr/bin/clickhouse"), "server"], b"ClickHouse server version 22.0.0.1 (official build).", (22, 0, 0, 1)),
+        ([Path("/usr/local/bin/clickhouse", "server")], b"ClickHouse server version 22.1.2.3.", (22, 1, 2, 3)),
+    ],
+)
+def test_get_clickhouse_version(command: List[Union[str, Path]], output: bytes, expected: tuple[int]):
+    with patch("subprocess.check_output", return_value=output):
```

Comment:
Instead of having to `mock.patch`, I think it would be better to have a `parse_clickhouse_version(command_output: bytes) -> Sequence[int]` and keep the `subprocess` outside of the parsing function.

Reply:
Will do, hesitated as it seemed overkill to split that.

Reply:
I think it's fine if the subprocess goes back into the original place without creating a third function, it's not like we use it more than once.